lambdas/db-provision-user-database/src/index.ts

```diff
@@ -83,6 +83,7 @@ export const handler = async (event: HandlerEvent): Promise<void> => {
         database: `${dbUser}_db`,
         host: (rootKnexConfig.connection as Knex.PgConnectionConfig).host,
         port: (rootKnexConfig.connection as Knex.PgConnectionConfig).port,
+        disableSSL: 'true',
```

Comment:
I had to add disableSSL: 'true' to this file in order for Bamboo CI in order for the Deploy Dev Integration Stack to complete successfully. Perhaps we need to add documentation around this addition?

Reply:
@Jkovarik This is the additional change I needed to make in order for Bamboo CI in order for the Deploy Dev Integration Stack to complete successfully

Reply:
We should probably consider making this a user option as part of the incoming event. 

That said, the behavior is documented as part of the ticket CL: 

```
- **CUMULUS-3323**
  - Added `disableSSL` as a valid database secret key - setting this in your database credentials will
    disable SSL for all Core database connection attempts.
  - Added `rejectUnauthorized` as a valid database secret key - setting
    this to `false` in your database credentials will allow self-signed certs/certs with an unrecognized authority.
  - Updated the default parameter group for `cumulus-rds-tf` to set `force_ssl`
    to 1.   This setting for the Aurora Serverless v1 database disallows non-SSL
    connections to the database, and is intended to help enforce security
    compliance rules.  This update can be opted-out by supplying a non-default
    `db_parameters` set in the terraform configuration.
 ```
 
 and 
 
 https://github.com/nasa/cumulus/pull/3587/files#diff-7b65a9166b23c12f9723277e9247262dceec1df189b6b67e3a471da1fe8316ab
 
 
That said - why do we need to disable SSL for v2?  Is this the cert concern, and if so can we leave SSL enabled but allow unverified certs?

Reply:
I have removed the disableSsl entry and added rejectUnauthorized = false. Bamboo CI ran successfully.